app/controllers/api/v1/friends_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Api::V1::FriendsController < ApplicationController
+  def index
+    if params[:request_status] == "accepted"
+      user = User.find_by(google_id: request.headers.env["HTTP_USER"])
```

Comment:
do we want to refactor this so we don't need as big of an if tree so it could be something like:

```
if params[:request_status].present?
  user = User.find_by(google_id: request.headers.env["HTTP_USER"])
  friends = user.friends_by_status(params[:request_status])
else
      user = User.find_by(google_id: request.headers.env["HTTP_USER"])
      all_friend_ids = user.all_friend_ids
      render json: UserSerializer.new(User.find(all_friend_ids))
    end
```

Reply:
Just updated this!